ocp_utilities/infra.py

```diff
@@ -81,6 +85,73 @@ def assert_pods_failed_or_pending(pods):
         )
 
 
+def assert_nodes_in_healthy_condition(
+    nodes,
+    bypass_check_if_kubelet_ready=True,
+    healthy_node_condition_type=None,
+):
+    """
+    Validates nodes are in a healthy condition.
+    Nodes Ready condition is True and none of the following node conditions is True:
+        - DiskPressure
+        - MemoryPressure
+        - PIDPressure
+        - NetworkUnavailable
+
+    Args:
+         nodes(list): List of Node objects
+
+         bypass_check_if_kubelet_ready:
+            Bypass checking a node's conditions if node.kubelet_ready is True
+
+         healthy_node_condition_type (dict):
+            Dictionary with condition type and the respective healthy condition status:
+            Example: {"DiskPressure": "False", ...}
+
+    Raises:
+        NodesNotHealthyConditionError: if any nodes DiskPressure
+        MemoryPressure, PIDPressure, NetworkUnavailable, etc condition is True
+    """
+    LOGGER.info("Verify all nodes are in a healthy condition.")
+
+    default_healthy_node_condition_type = {
```

Comment:
`default_healthy_node_condition_type`
Please define it only if not `healthy_node_condition_type`

Reply:
done